music_assistant/providers/audible/audible_helper.py

```diff
@@ -217,15 +333,42 @@ async def _call_api(self, path: str, **kwargs: Any) -> Any:
             )
         return response
 
-    async def _parse_audiobook(self, audiobook_data: dict[str, Any]) -> Audiobook:
+    async def _parse_audiobook(self, audiobook_data: dict[str, Any] | None) -> Audiobook:
+        if audiobook_data is None:
+            self.logger.error("Received None audiobook_data in _parse_audiobook")
+            # Return a minimal valid Audiobook to avoid breaking the iteration
```

Comment:
this feels weird, returning a non-existent item. That is going to bite you later.
Better just filter the invalid items out earlier in the process